runtime/gc_modron_startup/mmparseXXgc.cpp

```diff
@@ -1484,6 +1484,12 @@ gcParseXXgcArguments(J9JavaVM *vm, char *optArg)
 			continue;
 		}
 
+		if (try_scan(&scan_start, "noPreserveRemainders")) {
+
+			extensions->preserveRemainders = false;
+			continue;
+		}
+
```

Comment:
I understand this option is created for testing purpose. However should we add `preserveRemainders` for symmetry (for example in case we change default from `true` to `false`)?

Reply:
add preserveRemainders options in XXgc